rules/data-preprocessing.smk

```diff
@@ -145,70 +135,35 @@ rule raw_protected_areas:
 
 
 rule raw_srtm_elevation_tile_zipped:
-    message: "Download SRTM elevation data tile (x={wildcards.x}, y={wildcards.y}) from CGIAR."
+    message: "Download SRTM elevation data tile (tile={wildcards.tile}) from viewfinderpanoramas."
     output:
-        protected("data/automatic/raw-srtm/srtm_{x}_{y}.zip")
+        protected("data/automatic/raw-srtm/srtm_{tile}.zip")
     shell:
         """
-        curl -sLo {output} '{URL_CGIAR_TILE}/srtm_{wildcards.x}_{wildcards.y}.zip'
+        curl -sLo {output} '{URL_SRTM_TILE}/{wildcards.tile}.zip'
         """
 
 
-rule raw_srtm_elevation_tile:
-    message: "Unzip SRTM elevation data tile (x={wildcards.x}, y={wildcards.y})."
-    input:
-        "data/automatic/raw-srtm/srtm_{x}_{y}.zip"
-    output:
-        temp("build/srtm_{x}_{y}.tif")
-    run: # using Python here, because "unzip" issues a warning sometimes causing snakemake to break
-        import zipfile
-        from pathlib import Path
-
-        file_to_extract = Path(input[0]).with_suffix(".tif").name
-        with zipfile.ZipFile(input[0], "r") as zipref:
-            zipref.extract(file_to_extract, path="build")
-
-
-rule raw_srtm_elevation_data:
+rule elevation_in_europe:
     message: "Merge all SRTM elevation data tiles."
     input:
-        ["build/srtm_{x:02d}_{y:02d}.tif".format(x=x, y=y)
-         for x in range(SRTM_X_MIN, SRTM_X_MAX + 1)
-         for y in range(SRTM_Y_MIN, SRTM_Y_MAX + 1)
-         if not (x is 34 and y in [3, 4, 5, 6])] # these tiles do not exist
-    output:
-        temp("build/raw-srtm-elevation-data.tif")
-    conda: "../envs/default.yaml"
-    shell:
-        "rio merge {input} {output} --overwrite"
-
-
-rule raw_gmted_elevation_tile:
-    message: "Download GMTED elevation data tile."
-    output:
-        protected("data/automatic/raw-gmted/raw-gmted-{y}-{x}.tif")
-    run:
-        url = "{base_url}/{x_inverse}/{y}{x}_20101117_gmted_mea075.tif".format(**{
-            "base_url": URL_GMTED_TILE,
-            "x": wildcards.x,
-            "y": wildcards.y,
-            "x_inverse": wildcards.x[-1] + wildcards.x[:-1]
-        })
-        shell("curl -sLo {output} '{url}'".format(**{"url": url, "output": output}))
-
-
-rule raw_gmted_elevation_data:
-    message: "Merge all GMTED elevation data tiles."
-    input:
-        ["data/automatic/raw-gmted/raw-gmted-{y}-{x}.tif".format(x=x, y=y)
-         for x in GMTED_X
-         for y in GMTED_Y
-         ]
+        ["data/automatic/raw-srtm/srtm_{}.zip".format(idx)
+         for idx, bounds in
+         pd.read_csv("data/viewfinderpanoramas.csv", index_col=0, header=0).iterrows()
```

Comment:
Great idea! One comment though: the pathname should not be hardcoded. Instead, it should be a parameter in the config: `path-to-viewfinder-tiles` or something like that.